webpages/profile/metaanalysis.html

```diff
@@ -71,34 +74,60 @@ <h1>Are you sure to rename the meta-analysis from <span class="title"></span>?</
-      </div>
-      <div class="popupbox editing" data-boxid='reference'>
-        <div class="pin"></div>
-        <header><p>Reference</p></header>
-        <p class="description">Details of where this Meta-analysis has been published (if anywhere). Should include authors, year, title, and all the usual bibliographic information.</p>
-        <!-- todo comments -->
-      </div>
+
+    <div class="navbar">
+      <button class="tablinks active" onclick="openData(event, 'infoTab')">Info</button>
+      <button class="tablinks" onclick="openData(event, 'matTab')">Meta-analysis Table</button>
+      <button class="tablinks" onclick="openData(event, 'plotsTab')">Meta-analysis Plots</button>
+      <button class="tablinks" onclick="openData(event, 'aggTab')">Meta-analysis Aggregates</button>
     </div>
-    <p class="description edithighlight">
-      <span data-focuses=".value.editing">Description: </span>
-      <span class="value notediting">error</span>
-      <span class="value editing" contenteditable placeholder="add description">error</span>
-    </p>
+   
+
+    <!-- TODO ##### BASIC INFO #########-->
+
+    <div id="infoTab" class="tabcontent">
+      <div class="published popupboxtrigger">
+        <div class="edithighlight">
+          <span class="popupboxhighlight" data-focuses=".value.editing">Reference:</span>
+          <span class="value notediting">error</span>
+          <span class="value editing oneline" contenteditable placeholder="authors, year, title, journal etc.">error</span>
+        </div>
+        <div class="popupbox editing" data-boxid='reference'>
+          <div class="pin"></div>
+          <header><p>Reference</p></header>
+          <p class="description">Details of where this Meta-analysis has been published (if anywhere). Should include authors, year, title, and all the usual bibliographic information.</p>
+          <!-- todo comments -->
+        </div>
+      </div>
+      <p class="description edithighlight">
+        <span data-focuses=".value.editing">Description: </span>
+        <span class="value notediting">error</span>
+        <span class="value editing" contenteditable placeholder="add description">error</span>
+      </p>
+  </div>
+    <!-- TODO ############# END ################-->
+
+
     <!-- todo show "other contributors" or "forks" or "clones" ... -->
     <!-- todo show meta-analyses that use this paper -->
     <!-- todo show history -->
     <!-- when not editing, show something for empty doi and link (or nothing at all?) -->
     <!-- todo comments -->
-    <p class="ma-table-heading">Meta-analysis table:</p>
-    <div class="experiments"></div>
-    <div class="plots"></div>
-    <div class="graphs"></div>
-    <div class="aggregates"></div>
+    
+    <div id="matTab" class="tabcontent">
+      <p class="ma-table-heading">Meta-analysis table:</p>
+      <div class="experiments"></div>
+    </div>
+    <div id="plotsTab" class="plots tabcontent">
```

Comment:
the container of plots might be inside this div, like above the container for experiments, if we remove `>` from before `.plots` in metaanalyses.js in multiple places